Update docs/footer page content (#2202)
vanilla-framework/vanilla-framework: docs/en/patterns/footer.md

## Footer

<hr>

> You can use this component at the bottom of a document page or interface that contains data common to other pages. The information in footers, includes things like copyright notices, legal information or related links.

Reviewer: ```suggestion
You can use this component at the bottom of a document page or interface to contain data common to other pages. The suggested content for footers includes things such as copyright notices, legal information and related links.
```
